fplus-lib/src/core/mod.rs

```diff
@@ -1970,24 +1977,32 @@ impl LDNApplication {
         issue_number: String,
         info_owner: String,
         info_repo: String,
-        db_model: ApplicationModel,
+        db_model: Option<ApplicationModel>,
     ) -> Result<bool, LDNError> {
-        let gh = github_async_new(info_owner.clone(), info_repo.clone()).await;
-    
-        let comment = if db_model.owner == info_owner && db_model.repo == info_repo {
-            // Application already exists in the same repository
-            format!(
-                "This wallet address already exists in another application: #{}",
-                db_model.pr_number
-            )
-        } else {
-            // Application exists in a different repository
-            format!(
-                "This wallet address already exists in another application: http://github.com/{}/{}/issues/{}",
-                db_model.owner, db_model.repo, db_model.pr_number
-            )
-        };
+        let mut comment = "This wallet address has already received datacap in other application".to_string();
+
+        if let Some(db_model) = db_model {
+            //Load application from db_model.application string json
+            let application = ApplicationFile::from_str(&db_model.application.unwrap())
+                .map_err(|e| LDNError::Load(format!("Failed to parse application file from DB: {}", e)))?;
+        
+            comment = if db_model.owner == info_owner && db_model.repo == info_repo {
+                // Application already exists in the same repository
+                format!(
+                    "This wallet address already exists in another application: #{}",
+                    application.issue_number
+                )
+            } else {
+                // Application exists in a different repository
+                format!(
+                    "This wallet address already exists in another application: http://github.com/{}/{}/issues/{}",
```

Comment:
```suggestion
                    "This client address has also applied for datacap at http://github.com/{}/{}/issues/{} - Please use a new, distinct client address for this application, so that usage can be clearly understood as relating to this application.",
```